### PR TITLE
#winningest_coach/#worst_coach methods

### PR DESCRIPTION
**Pull request by zach-bergman**
## Type Of Change
- [X] New Feature
- [X] Debugging
- [X] Refactor/File Clean-up
- [X] Tests
- [ ] Documentation

## Provide A Summary Of Your Change
#winningest_coach and #worst coach methods completed. Followed TDD. Also ran methods in spec harness and passed.

## Group Project Check List
- [X] I made test for my code, and made sure they were passing before sucmitting PR
- [X] I assigned atleast 2 group members to review my PR

### CODE REVIEW

File: lib/game_team.rb
Comment (noahdurbin) [inline lib/game_team.rb:1]: these methods look great, looks like some of this was pretty complicated to figure out. Great naming on your methods here.

File: spec/game_team_spec.rb [hunk 1]
```diff
@@ -20,29 +20,108 @@
         takeaways: 7
     }
     @gameteam_attr_test = GameTeam.new(gameteam_data)
```
Comment (noahdurbin) [inline spec/game_team_spec.rb:22]: I like the way you did this

File: spec/game_team_spec.rb [hunk 2]
```diff
@@ -20,29 +20,108 @@
...
+          expect(@gameteam_attr_test.pim).to eq(8)
+          expect(@gameteam_attr_test.ppo).to eq(3)
+          expect(@gameteam_attr_test.ppg).to eq(0)
+          expect(@gameteam_attr_test.faceoff_win_per).to eq(44.8)
+          expect(@gameteam_attr_test.giveaways).to eq(17)
+          expect(@gameteam_attr_test.takeaways).to eq(7)
+      end
+    end
+
+  describe "GameTeam Class Methods" do
```
Comment (noahdurbin) [inline spec/game_team_spec.rb:49]: Great testing here.

**Review by noahdurbin — APPROVED** [review]

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -1,17 +1,17 @@
 require 'simplecov'
 require 'csv'
 require 'pry'
+require "./lib/stat_tracker"
```
Comment (austincarrjones) [inline spec/spec_helper.rb:4]: thanks for thinking through the stat_tracker architecture!

**Review by austincarrjones — APPROVED** [review]

**Pull request merged**